### PR TITLE
Added home screen with view of all images

### PR DESCRIPTION
**Pull request by sukhada**
@caro5

### CODE REVIEW

File: app/src/main/java/turtler/voyageur/activities/BaseActivity.java [hunk 1]
```diff
@@ -75,6 +87,10 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         ButterKnife.bind(this);
         setSupportActionBar(mToolbar);
+        gridView = (GridView) findViewById(R.id.gvImageGrid);
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/activities/BaseActivity.java]: butterknife is imported fyi

File: app/src/main/java/turtler/voyageur/adapters/ImageGridAdapter.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.List;
+
+import turtler.voyageur.R;
+import turtler.voyageur.models.Image;
+import turtler.voyageur.models.ImageItem;
+
+/**
+ * Created by skulkarni on 9/13/16.
+ */
+public class ImageGridAdapter extends ArrayAdapter {
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/adapters/ImageGridAdapter.java:22]: can we just create an imageadapter based on the Parse Image class? It'll be easier in the future to connect an image to an event or a trip or something

**Comment by sukhada** [comment]
@caro5 done

**Comment by caro5** [comment]
lgtm 🍟

**Pull request merged**